tool_collections/kraken/kraken/README.md

```diff
@@ -0,0 +1,25 @@
+How to test Kraken in your local galaxy instance:
+
+In your galaxy home directory config/tool_conf.xml file, change  
+
+```
+ <section name="Kraken-test" id="kraken">
```

Comment:
@davebx is this needed, an old left over? I have done this as well, but this is was way before TS :)